test/keys_manager_test.js

```diff
@@ -289,6 +289,21 @@ contract('KeysManager [all features]', function (accounts) {
       logs[0].args.key.should.be.equal(accounts[2]);
       logs[0].args.action.should.be.equal('added');
     })
+    it('cannot add removed mining key', async () => {
```

Comment:
Let's add one more test `cannot swap to removed mining key`.

Reply:
Sure